### DISCUSSION TITLE
Json column to text

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2916] [Q&A]
How to convert json column to text?

I am using view collection. My query doesn't convert json column to text.

```sql
CAST(branch as TEXT) 

```

I wrote my query like this but not working.

**Comment by ganigeorgiev** [comment]
I'll consider it in the future, but currently this is not supported.

Usually expressions, including `CAST`, are resolved to a `json` field. The only exception to this at the moment are some commonly used numeric expressions that loosely matches the conditions listed in https://github.com/pocketbase/pocketbase/blob/b9993aaa738eb7a0139b46034f676fc9ceb79bfb/daos/view.go#L260-L276.
